src/components/input/Checkbox.tsx

```diff
@@ -39,7 +39,7 @@ export type CheckboxProps = CompositeProps &
 export default function Checkbox({
   checked,
   defaultChecked = false,
-  icon = CheckboxOutlineIcon,
+  icon = CheckboxIcon,
   checkedIcon = CheckboxCheckedIcon,
```

Comment:
I'd suggest we take the opportunity and set `CheckboxCheckedFilledIcon` as the default `checkedIcon`

Reply:
Updated